pkg/net/cidrs/aws/internal/ranges2go/main.go

```diff
@@ -27,11 +27,7 @@ func main() {
 		outputPath = "./zz_generated_range_data.go"
 	}
 	// parse raw AWS IP range data
-	parsed, err := parseIPRangesJSON([]byte(ipRangesRaw))
-	if err != nil {
-		panic(err)
-	}
-	rtp, err := regionsToPrefixesFromData(parsed)
+	rtp, err := regionsToPrefixesFromRaw(ipRangesRaw)
```

Comment:
we want to minimize the amount of logic in any main.go, and ensure we're testing everything else thoroughly with quick and cheap unit tests ...

Reply:
ack